image.go

```diff
@@ -0,0 +1,147 @@
+package koebiten
+
+import (
+	"image/color"
+	"io/fs"
+	"math"
+
+	"tinygo.org/x/drivers/image/png"
+	"tinygo.org/x/drivers/pixel"
+)
+
+var _ Displayer = (*Image)(nil)
+
+// Image is a Wrapper for the pixel.Image type.
+//
+// Image implements the Displayer interface.
+type Image struct {
+	img pixel.Image[pixel.Monochrome]
+}
+
+// Size returns the width and height of the image.
+//
+// It implements the Displayer interface.
+func (i *Image) Size() (int16, int16) {
+	x, y := i.img.Size()
+	return int16(x), int16(y)
+}
+
+// SetPixel sets the pixel at the given x and y coordinates to the given color.
+// The color is converted to a pixel.Monochrome color.
+//
+// It implements the Displayer interface.
+func (i *Image) SetPixel(x, y int16, c color.RGBA) {
+	i.img.Set(int(x), int(y), pixel.NewMonochrome(c.R, c.G, c.B))
```

Comment:
It’s probably best to add **boundary checks**.  
There are cases where **out-of-bounds access** can occur when scaling or rotating.

> panic: Image.Set: out of bounds